jobs/cache_maintenance/src/cache_maintenance/config.py

```diff
@@ -30,29 +30,40 @@ def from_env(cls) -> "BackfillConfig":
 
 
 DUCKDB_INDEX_CACHE_DIRECTORY = None
-DUCKDB_INDEX_SUBDIRECTORY = "downloads"
-DUCKDB_INDEX_EXPIRED_TIME_INTERVAL_SECONDS = 10 * 60  # 10 minutes
-DUCKDB_INDEX_FILE_EXTENSION = ".duckdb"
+DUCKDB_INDEX_DOWNLOADS_SUBDIRECTORY = "downloads"
+DUCKDB_INDEX_JOB_RUNNER_SUBDIRECTORY = "job_runner"
```

Comment:
nit:

move these hardcoded values to libcommon.constants?

Reply:
done